shopelectro/settings/base.py

```diff
@@ -38,8 +38,12 @@
 # https://docs.djangoproject.com/en/1.9/ref/settings/#secure-proxy-ssl-header
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
-# Application definition
+# Enable in frame loading for Ya.Metric
+# https://docs.djangoproject.com/es/1.10/ref/clickjacking/
```

Comment:
@YozhEzhi , y, we can, but we'll switch to 1.10 soon. FullTextSearch feature requires this.
And this link have no specific of 1.9 or 1.10 version